### DISCUSSION TITLE
Contributions not showing up on graph, no matter what I do

### DISCUSSION THREAD

**Discussion by thewebmasterp** [community/community#23061] [New to GitHub]
<p>Hi. Suddenly, my contributions stopped showing up on my contribution graph. I checked both emails, the one I have set as default in my github account as well as git’s user.email. The strange thing is that this happened since I added a secondary email which was not set as default. From that moment, all contributions stopped showing up. I tried removing this secondary email to no avail.</p>
<p>I haven’t changed anything about my repos branches. Tried committing and pushing old repos which I know that should show up on the graph, not working.</p>
<p>Thanks in advance!</p>

**Comment by jfullstackdev** [comment]
<p>just a quick check, your commits are unverified from the latest repo I opened</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/2/322990c536db2d4d2c109d15968d5449528d5036.png" data-download-href="/uploads/short-url/79KVudmZCnCYUD4akgjJnPKEJJs.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074640-bced458e-37a7-4fca-9343-e40fbcf1c955.png" alt="image" data-base62-sha1="79KVudmZCnCYUD4akgjJnPKEJJs" width="690" height="394" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/322990c536db2d4d2c109d15968d5449528d5036_2_690x394.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/2/322990c536db2d4d2c109d15968d5449528d5036_2_1035x591.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/2/322990c536db2d4d2c109d15968d5449528d5036_2_1380x788.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/322990c536db2d4d2c109d15968d5449528d5036_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1567×896 53.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>it can result to irregularities, particularly when you add another email</p>

**Comment by thewebmasterp** [comment]
<p>Hi. Thanks for the help!</p>
<p>How aren’t they verified? I can’t see any button or anything to verify them. Is it a problem that they’re in the “main” branch (which is the only branch, the default one) but not in “master”?</p>

**Comment by thewebmasterp** [comment]
<p>Update: Creating a new repository is shown on the contributions graph but later committing to it isn’t. An example is the <a href="https://github.com/thewebmasterp/test-repo" rel="noopener nofollow ugc">test-repo</a> that I created.</p>
<p>Why creating the repo shows on the contributions graph but later the commits aren’t being shown on the graph?</p>
<p>Thanks.</p>

**Comment by thewebmasterp** [comment, marked as answer]
<p>Issue solved. Typo in my git config --global user.email returns <a href="mailto:thewebmsaterp@gmail.com">thewebmsaterp@gmail.com</a> whereas my actual email is <a href="mailto:thewebmasterp@gmail.com">thewebmasterp@gmail.com</a>. Thanks and sorry for the spam…</p>
